lib/mapnik_backend.js

```diff
@@ -226,57 +201,50 @@ MapnikSource.registerProtocols = function(tilelive) {
 // can be expected to be in `mapfile`. If this isn't successful, `callback` gets
 // an error as its first argument.
 MapnikSource.prototype._loadXML = function(callback) {
-    var source = this;
     this._base = path.resolve(path.dirname(this._uri.pathname));
 
+
     // This is a string-based map file. Pass it on literally.
     if (this._uri.xml) return callback(null, this._uri.xml);
 
     // Load XML from file.
-    fs.readFile(path.resolve(source._uri.pathname), 'utf8', function(err, xml) {
-        if (err) return callback(err);
-        callback(null, xml);
-    });
+    fs.readFile(path.resolve(this._uri.pathname), 'utf8', callback);
```

Comment:
That's cool, but we never reach this code 👿. We always use `this._uri.xml`.

Reply:
FYI

Reply:
I know, but just in case we decide to change it in the future.